### ISSUE TITLE
onSwiping TypeScript definition is wrong

### ISSUE THREAD

**Issue by mi5ha** [webraptor/react-native-deck-swiper#44]
In index.d.ts, onSwiping is defined as:

```
onSwiping?: () => void;
```

I think it should be?

```
onSwiping?: (x: number, y: number) => void;
```

**Comment by RuzenDev** [comment]
Yeaah Somebody Release the version with this fix

**Comment by webraptor** [comment]
Will be fixed by 2.0.10
